tests/FSharpPlus.Tests/General.fs

```diff
@@ -872,6 +873,39 @@ module Foldable =
         areEqual ["Using WrappedListA's Length"; "Using WrappedListD's Length"] (SideEffects.get ())
         ()
 
+    [<Test>]
+    let tryLast () =
+        let s                = tryLast <| seq [1;2]
+        let s': int option   = tryLast <| seq []
+        areEqual s (Some 2)
+        areEqual s' None
+
+        let l                = tryLast [1;2;3]
+        let l': int option   = tryLast []
+        areEqual l (Some 3)
+        areEqual l' None
+
+        let a                = tryLast [|1|]
+        let a': int option   = tryLast [||]
+        areEqual a (Some 1)
+        areEqual a' None
+
+        let nes              = tryLast <| NonEmptySeq.ofList [1;2]
+        areEqual nes (Some 2)
+
+        let i                = tryLast <| Id.create 4
+        areEqual i (Some 4)
```

Comment:
`Id` is a dummy type we use internally to create ambiguity in the overload resolution, so better remove it from the tests, it could create confusion and there is not point at all in testing it.

Reply:
Ok, removed